src/client/app/containers/view.mode.ts

```diff
@@ -45,6 +53,27 @@ export class ViewMode implements OnInit, OnDestroy {
         return environment.current.host.toLowerCase();
     }
 
+    get isGist() {
+        return this.snippet && this.snippet.gist;
+    }
+
+    get openInPlaygroundSupported() {
+        let host = environment.current.host.toUpperCase();
+        return host === HostType.EXCEL || host === HostType.WORD || host === HostType.POWERPOINT;
+    }
+
+    get openInHostString() {
+        return this.strings.openInHost.replace('{0}', environment.current.host);
+    }
+
+    get downloadAsHostFileString() {
+        return this.strings.downloadAsHostFile.replace('{0}', environment.current.host);
+    }
+
+    get urlString() {
+        return `URL: ${window.location.href}`;
```

Comment:
I think it's correct as is.  I want the URL to be something that the users can copy-paste to get the same snippet in full-screen mode.  And the "open in github" to take them to github.